docs/04_web/02_web-components/01_form/10_select.md

```diff
@@ -1,46 +1,46 @@
----
-title: 'Web Components - Select'
-sidebar_label: 'Select'
-id: select
-keywords: [web, web components, select]
-tags:
-    - web
-    - web components
-    - select
----
-
-<div class="select-examples">
-
-An implementation of an [HTML select element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) as a form-connected Web Component.
-
-## Set-up
-
-```ts
-import { provideDesignSystem, alphaSelect, alphaOption } from '@genesislcap/alpha-design-system';
-
-provideDesignSystem().register(alphaSelect(), alphaOption());
-```
-
-## Usage
-
-```html live
-<alpha-select id="shirt-size">
-  <alpha-option value="s">Small</alpha-option>
-  <alpha-option value="m">Medium</alpha-option>
-  <alpha-option value="l">Large</alpha-option>
-  <alpha-option value="xl">Extra Large</alpha-option>
-</alpha-select>
-```
-By default, 'Select' only allows the user to select one of the options defined. When a user selects an option, it automatically removes the existing selection. 
-
-If you want to allow the user to make multiple selections, add the keyword 'multiple' to the component.
-
-## Use cases
-
-- selecting from a list of options
-
-## Additional resources
-
-- [W3C Component Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/#Listbox)
-
-</div>
+---
+title: 'Web Components - Select'
+sidebar_label: 'Select'
+id: select
+keywords: [web, web components, select]
+tags:
+    - web
+    - web components
+    - select
+---
+
+<div class="select-examples">
+
+An implementation of an [HTML select element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) as a form-connected Web Component.
+
+## Set-up
+
+```ts
+import { provideDesignSystem, alphaSelect, alphaOption } from '@genesislcap/alpha-design-system';
+
+provideDesignSystem().register(alphaSelect(), alphaOption());
+```
+
+## Usage
+
+```html live
+<alpha-select id="shirt-size">

```

Comment:
https://github.com/genesiscommunitysuccess/docs/assets/43502076/f7943c94-a85d-482e-b66e-d616f567954f

When I add the attribute the behaviour seems wrong

Reply:
Created a sepate [jira](https://genesisglobal.atlassian.net/browse/FUI-1326) to address this.